client/containers/DashboardOverview/helpers/RecentVisitList.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { Icon } from 'components';
+import { UserScopeVisit, Pub, Collection } from 'utils/types';
+import { getDashUrl } from 'utils/dashboard';
+
+require('./recentVisitList.scss');
+
+type Props = {
+	userScopeVisits: UserScopeVisit[];
+	pubs: Pub[];
+	collections: Collection[];
+};
+
+const RecentVisitList = (props: Props) => {
+	const { userScopeVisits, pubs, collections } = props;
+	const recentVisits = userScopeVisits
```

Comment:
A couple of minor thoughts here:

1. Can you break this calculation into a function for cleanliness' sake?
2. When doing that it would be nice to define a type based on the `{href, iconName, id, title}` shape you're using